tests/schemas/api_v3/networkStatistics.schema.js

```diff
@@ -15,25 +15,27 @@
  */
 import Joi from 'joi';
 
-const goodRequestSchema = {
-	data: Joi.object().required(),
-	meta: Joi.object().required(),
-	links: Joi.object().optional(),
-};
+const regex = require('./regex');
 
 const basicStatsSchema = {
 	totalPeers: Joi.number().integer().min(0).required(),
 	connectedPeers: Joi.number().integer().min(0).required(),
 	disconnectedPeers: Joi.number().integer().min(0).required(),
 };
 
+const totalItem = {
+	amount: Joi.string().pattern(regex.POSITIVE_DIGITS).required(),
+	tokenID: Joi.string().pattern(regex.TOKEN_ID).required(),
+};
+
 const networkStatisticsSchema = {
 	basic: Joi.object(basicStatsSchema).required(),
 	height: Joi.object().required(),
 	networkVersion: Joi.object().required(),
+	totalLocked: Joi.array().items(Joi.object(totalItem).required()).required(),
```

Comment:
```suggestion
	totalLocked: Joi.array().items(Joi.object(totalItem).required()).min(1).required(),
```

Reply:
Since we are using `required` check for the items, it will already validate existance of atleast one element.